components/SpotlightShare/SpotlightShare.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent } from 'react';
+import Modal from 'react-modal';
+import { Button } from '../Button';
+import classNames from 'classnames';
+import { FaFacebookF, FaTwitter } from 'react-icons/fa';
+import { MdEmail } from 'react-icons/md';
+import './styles.css';
+
+interface SpotlightShareProps {
+  maxHeight?: string;
+  minHeight?: string;
+  className?: string;
+}
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    width: '50%',
+    height: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)'
+  }
+};
+
+const SpotlightShare: FunctionComponent<SpotlightShareProps> = props => {
+  const [modalIsOpen, setIsOpen] = React.useState(false);
```

Comment:
Import `useState` through destructuring. `...{ useState } from 'react'`, that way you don't need to `React.useState`

Name your state properties conventionally ... if your property is `isLoading`, the modifier should be `setIsLoading` ... that way, when you encounter it in code, you know what it's modifying. `setIsOpen` will seem like it's modifying a variable called `isOpen`.